enterprise-suite/console_test/prometheus_test.go

```diff
@@ -0,0 +1,264 @@
+package console
+
+import (
+	"fmt"
+
+	"github.com/lightbend/console_test/util"
+	"github.com/lightbend/console_test/util/kube"
+	"github.com/lightbend/console_test/util/monitor"
+	"github.com/lightbend/console_test/util/prometheus"
+
+	apiv1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+const promTestNamespace = "smoketest-prometheus"
```

Comment:
This can't be hardcoded - because it will vary based on environment we're deploying, such as with openshift.

I'm thinking we shouldn't specify it at all. Instead we should rely on the kubectl context being set-up to point to the correct namespace (which it does currently in travis).

Reply:
Removed this, now all testing happens in `lightbend-test` which is also overridable with `--namespace` flag.